src/styles/typography.css

```
@@ -322,16 +322,21 @@
       @apply pl-0;
 
       ol {
-        @apply mt-3.5 pl-4;
+        @apply mt-3.5 list-none pl-7;
         counter-reset: main-counter;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)